src/libdrakvuf/libdrakvuf.h

```diff
@@ -182,15 +182,22 @@ typedef enum memaccess_type {
     POST
 } memaccess_type_t;
 
+typedef struct process_data {
+    char *name;         /* Process name */
```

Comment:
This should be `const` otherwise the a plugin may erroneously end up freeing it.

Reply:
I initially defined that way but surprisingly gcc complained a lot with the following errors:
```error: passing argument 1 of 'g_free' discards 'const' qualifier from pointer target type```

...so I ended up getting rid of the const qualifier instead of doing a cast. :)

Reply:
The way I got around that is to have a `char *` version of procname within libdrakvuf that can bee freed without warnings but a `const char*` version is getting passed to the plugins through the info structure.

Reply:
I'm compiling with debug enabled by the way...

Reply:
...but now instead of having just one field (```procname```) defined in ```private.h``` and in ```libdrakvuf.h``` we have a ```proc_data_t``` structure with some fields defined within so I don't see an easy way to get rid of this without repeating the structure definition.

I think I'm going to make all the casts neded... :)